src/main/java/com/github/resalner/javapractice/dto/OrderItemResponse.java

```diff
@@ -1,18 +1,8 @@
 package com.github.resalner.javapractice.dto;
 
-import lombok.Data;
-import lombok.NoArgsConstructor;
-import lombok.AllArgsConstructor;
-
-import java.util.List;
-
-@Data
-@AllArgsConstructor
-@NoArgsConstructor
-public class OrderItemResponse {
-    private long id;
-    private long productId;
-    private long orderId;
-    private Integer count;
-    private double price;
-}
\ No newline at end of file
+public record OrderItemResponse(long id,
```

Comment:
Во всех record-ах поправь, чтобы были такие отступы
```suggestion
public record OrderItemResponse(
        long id,
```